examples/llama2/user_script.py

```diff
@@ -56,15 +56,18 @@ def get_merged_sample_with_past_kv_inputs(
     # By using a single function with no default values, we can avoid confusion and are deliberate about the sizes
     # can instead write dummy input functions like 'get_merged_decoder_with_past_dummy_inputs' if needed
 
-    if model_id:
+    if model:
+        config = model.load_model().config
+    elif model_id:
         config = LlamaConfig.from_pretrained(model_id)
     else:
         config = LlamaConfig.from_pretrained(model.hf_config.model_name)
```

Comment:
I think we can cover the required changes using `config = model.get_hf_model_config()` at line 64. This uses the `model_path` to load the llamaconfig without having to load the model entirely. 

Otherwise, the else condition never gets used since it's supposed to be the same condition as the new if.

Reply:
Done!